Build failes with features = [ "stm32h743", "smps" ]

When configured for [ "stm32h743", "smps" ], compilation fails with:
error[E0599]: no method named `sden` found for struct `stm32h7::R<u32, Reg<u32, stm32h7::stm32h743::pwr::_CR3>>` in the current scope
   --> /home/dbg/.cargo/registry/src/github.com-1ecc6299db9ec823/stm32h7xx-hal-0.9.0/src/pwr.rs:176:44
    |
176 |                 assert!(self.rb.cr3.read().sden().bit_is_clear(), error);
    |                                            ^^^^ method not found in `stm32h7::R<u32, Reg<u32, stm32h7::stm32h743::pwr::_CR3>>`

Versions: 
stm32h7xx-hal v0.9.0
stm32h7 v0.13.0

The reason for the failure is that the PWR_CR3 in stm32h743 does not have the SDEN bit.

--- a/src/pwr.rs
+++ b/src/pwr.rs
@@ -54,6 +54,9 @@ use crate::stm32::PWR;
 #[cfg(all(feature = "revision_v", not(feature = "rm0455")))]
 use crate::stm32::{RCC, SYSCFG};
 
+#[cfg(all(feature = "rm0433", feature = "smps"))]
+compile_error!("SMPS configuration fields are not available for RM0433 parts");
+
 /// Extension trait that constrains the `PWR` peripheral
 pub trait PwrExt {
     fn constrain(self) -> Pwr;

